app/settings/surveys/targeted-surveys/targeted-survey-edit.html

```diff
@@ -19,108 +19,79 @@ <h1 class="mode-context-title">Surveys</h1>
     </span>
 
     <div class="mode-context-body">
-
     </div>
 </div>
 
-
 <main role="main">
     <div class="toolbar">
-        <div class="button-group">
-            <button type="button" class="button-flat" ng-click="cancel()" translate>app.cancel</button>
-            <button type="submit" class="button-alpha"  ng-if="!saving_survey" ng-click="saveSurvey()">{{save}}</button>
-            <button type="submit" class="button-alpha"  disabled ng-if="saving_survey">{{saving}}
-              <div class="loading">
-                <div class="line"></div>
-                <div class="line"></div>
-                <div class="line"></div>
-              </div>
-          </button>
-        </div>
     </div>
+
     <div class="main-col">
-        <div class="form-sheet">
+        <div ng-show="targetedSurveysEnabled" class="form-sheet">
             <div class="form-sheet-summary">
                 <h2 class="form-sheet-title">Create a new targeted survey</h2>
                 <p>Send questions via SMS to a list of those phone numbers.</p>
             </div>
-            <div class="stepper" data-accordion-items="stepper-item">
-                <div class="stepper-item active">
+
+            <div class="stepper">
+                <div class="stepper-item" ng-class="{'active': isActiveStep(1), 'disabled': !isActiveStep(1)}">
                     <h2 class="stepper-heading" data-accordion-trigger>
                         <span class="stepper-badge">1</span>
                         Name your survey
```

Comment:
Oh, need to add those strings to en.json

Reply:
Will add these in my next PR.